src/containers/Tools.vue

```diff
@@ -56,13 +79,44 @@ export default {
   }
 
   &-cards {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: flex-start;
-  }
+    display: grid;
+    grid-auto-columns: min-content;
+    //grid-auto-rows: min-content;
+
+    column-gap: 5rem;
+    row-gap: 3rem;
+
+    > :nth-child(1) {
+      grid-column: 1/2;
+      grid-row: 1/2;
+    }
+
+    > :nth-child(3) {
+      grid-column: 2/3;
+      grid-row: 1/2;
+    }
+
+    > :nth-child(2) {
+      grid-column: 1/2;
+      grid-row: 2/3;
+    }
+
+    > :nth-child(4) {
+      grid-column: 2/3;
+      grid-row: 2/3;
+    }
```

Comment:
What is the purpose of this styling? I think it should be documented here.

Reply:
grid setup… i could just do individual classes for each tool card but i wanted to keep it more general. i'll prbly use individual classes to be cleaner